pkg/sdk/testint/security_integrations_gen_integration_test.go

```diff
@@ -429,26 +595,267 @@ func TestInt_SecurityIntegrations(t *testing.T) {
-		assert.Contains(t, details, sdk.SecurityIntegrationProperty{Name: "EXTERNAL_OAUTH_ALLOWED_ROLES_LIST", Type: "List", Value: role1.Name, Default: "[]"})
+		assertApiAuth(details, apiAuthDetails{
+			enabled:                    "true",
+			oauthAccessTokenValidity:   "31337",
+			oauthRefreshTokenValidity:  "31337",
+			oauthClientId:              "foo",
+			oauthClientAuthMethod:      string(sdk.ApiAuthenticationSecurityIntegrationOauthClientAuthMethodClientSecretPost),
+			oauthAuthorizationEndpoint: "http://example.com",
+			oauthTokenEndpoint:         "http://example.com",
+			oauthGrant:                 string(sdk.ApiAuthenticationSecurityIntegrationOauthClientGrantClientCredentials),
+			authType:                   "OAUTH2",
+			comment:                    "foo",
+		})
+
+		unsetRequest := sdk.NewAlterApiAuthenticationAuthorizationCodeGrantFlowSecurityIntegrationRequest(id).
+			WithUnset(
+				*sdk.NewApiAuthenticationAuthorizationCodeGrantFlowIntegrationUnsetRequest().
+					WithEnabled(true).
+					WithComment(true),
+			)
+		err = client.SecurityIntegrations.AlterApiAuthenticationAuthorizationCodeGrantFlow(ctx, unsetRequest)
+		require.NoError(t, err)
+
+		details, err = client.SecurityIntegrations.Describe(ctx, id)
+		require.NoError(t, err)
+
+		assert.Contains(t, details, sdk.SecurityIntegrationProperty{Name: "ENABLED", Type: "Boolean", Value: "false", Default: "false"})
+		assert.Contains(t, details, sdk.SecurityIntegrationProperty{Name: "COMMENT", Type: "String", Value: "", Default: ""})
+	})
+
+	t.Run("AlterApiAuthenticationAuthorizationCodeGrantFlow - set and unset tags", func(t *testing.T) {
+		tag, tagCleanup := testClientHelper().Tag.CreateTag(t)
+		t.Cleanup(tagCleanup)
+
+		_, id := createApiAuthCodeGrant(t, nil)
+
+		tagValue := "abc"
+		tags := []sdk.TagAssociation{
+			{
+				Name:  tag.ID(),
+				Value: tagValue,
+			},
+		}
+		alterRequestSetTags := sdk.NewAlterApiAuthenticationAuthorizationCodeGrantFlowSecurityIntegrationRequest(id).WithSetTags(tags)
+
+		err := client.SecurityIntegrations.AlterApiAuthenticationAuthorizationCodeGrantFlow(ctx, alterRequestSetTags)
+		require.NoError(t, err)
+
+		returnedTagValue, err := client.SystemFunctions.GetTag(ctx, tag.ID(), id, sdk.ObjectTypeIntegration)
+		require.NoError(t, err)
+
+		assert.Equal(t, tagValue, returnedTagValue)
+
+		unsetTags := []sdk.ObjectIdentifier{
+			tag.ID(),
+		}
+		alterRequestUnsetTags := sdk.NewAlterApiAuthenticationAuthorizationCodeGrantFlowSecurityIntegrationRequest(id).WithUnsetTags(unsetTags)
+
+		err = client.SecurityIntegrations.AlterApiAuthenticationAuthorizationCodeGrantFlow(ctx, alterRequestUnsetTags)
+		require.NoError(t, err)
+
+		_, err = client.SystemFunctions.GetTag(ctx, tag.ID(), id, sdk.ObjectTypeIntegration)
+		require.Error(t, err)
+	})
+	t.Run("AlterApiAuthenticationJwtBearerFlow", func(t *testing.T) {
```

Comment:
nit-pick: we use interlines between the tests

Reply:
Fixed.